lgc/include/lgc/patch/FragColorExport.h

```diff
@@ -53,11 +54,13 @@ enum class CompSetting : unsigned {
 // Represents the manager of fragment color export operations.
 class FragColorExport {
 public:
-  FragColorExport(PipelineState *pipelineState, llvm::Module *module);
+  FragColorExport(PipelineState *pipelineState, LgcContext *lgcContext, ColorExportState colorExportState);
```

Comment:
I will want to remove the pipeline state from this, so that the color export shader can be generated without a pipeline state object.  It still has one use.

Reply:
Sounds good to me as a path forward.